packages/paste-core/components/blockquote/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+`
```

Comment:
such a nitpick but did this happen automatically from the plop template?

Reply:
+1

Should be removed as t would show up on the docs page for the component